src/shared-components/container/index.js

```diff
@@ -4,11 +4,12 @@ import {
   Container,
   Section,
   Divider,
-  Image,
+  Image, VerticalDivider,
```

Comment:
Need to move this to a new line. Weird that eslint didn't catch it.

Reply:
That is bizarre; I feel like I did put it on its own line originally, but I must be mistaken. Fixed in fc3cce7f0a2862631ffa28c2fff08b64155cbb5e.